php_phongo.c

```diff
@@ -2491,7 +2491,7 @@ static void php_phongo_handshake_data_append(const char* name, size_t name_len,
 	char*  full_platform;
 
 	php_version_string_len = strlen(PHP_VERSION) + PHONGO_METADATA_PHP_VERSION_PREFIX_LEN + 1;
-	php_version_string     = ecalloc(sizeof(char*), php_version_string_len);
+	php_version_string     = ecalloc(php_version_string_len, sizeof(char*));
```

Comment:
Similar to the previous comment, I think the second arg should be `sizeof(char)` here.

Reply:
Correct.

```suggestion
	php_version_string     = ecalloc(php_version_string_len, sizeof(char));
```